src/api/app/views/webui/shared/_error.html.haml

```diff
@@ -0,0 +1 @@
+= flash[:error]
```

Comment:
Why is this shared partial needed? We already have [app/views/layouts/webui/_flash.html.haml](https://github.com/openSUSE/open-build-service/blob/775d283116ba0eec1e3f0647f42db75bc881a824/src/api/app/views/layouts/webui/_flash.html.haml) which is handling flash messages.

Reply:
True, removing